packages/angular/src/tree/tree.component.ts

```diff
@@ -136,7 +135,7 @@ const localDir = `${Stratus.BaseUrl}${boot.configuration.paths[`${systemDir}/*`]
  * @title Tree with Nested Drag & Drop
  */
 @Component({
-    selector: `sa-${moduleName}`,
+    selector: `sa-internal-${moduleName}`,
```

Comment:
This selector change may also cause an issue.  I'll have to check the changes on a fresh live test.